.github/workflows/build-and-test-all.yml

```diff
@@ -278,6 +279,7 @@ jobs:
       - run: inv test-api auth -b ${{ matrix.backend }}
       - run: inv generate-coverage-info /opt/pdns-auth/sbin/pdns_server $GITHUB_WORKSPACE
```

Comment:
For `recursor` and `dnsdist` jobs, this step is also skipped. Should we also add here `if: ${{ env.COVERAGE == 'yes' }}`?